website/content/docs/concepts/domain-model/credential-stores.mdx

```diff
@@ -0,0 +1,166 @@
+---
+layout: docs
+page_title: Domain Model - Credential Stores
+description: |-
+  The anatomy of a Boundary credential store
+---
+
+# Credential Stores
+
+A credential store is a resource
+that can retrieve, store, and potentially generate [credentials][]
+of differing types and differing access levels.
+It belongs to a [project][] and supports the
+[principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege)
+by providing mechanisms to limit the [credentials][] it can access
+to the minimum necessary for the project it is in.
+A credential store can also contain [credential libraries][].
+
+## Attributes
+
+A credential store has the following configurable attributes:
```

Comment:
ScopeId?

Reply:
We don't include parent IDs in the other domain docs. Those are documented in the API docs.